### PR TITLE
Update todo-tutorial-repository.md

### PR DESCRIPTION
**Pull request by SzymonSel**
Update src example to newest complaint version

## Checklist

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [x] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

**Comment by virkt25** [comment]
@SzymonSel Can you please amend the commit message to pass commit linting. 

We follow the conventional commits standard as described here: https://github.com/strongloop/loopback-next/blob/master/docs/site/DEVELOPING.md#commit-message-guidelines

TLDR -- Do the following
```
git commit --amend
```
Change message to something like:
```
docs: update todo-repository-tutorial.md
```

Push to your branch. Shoudl re-trigger CI. Once it's all Green I can merge it for you.

**Comment by virkt25** [comment]
@SzymonSel I rebased the PR for you and landed it. Thanks for your first contribution to LoopBack and welcome to the community. 🎉 Look forward to seeing more PRs if you have time 😉

**Comment by SzymonSel** [comment]
Ah yes, thanks. I will definitely try to chip in more :)
